.github/workflows/docs.yml

```diff
@@ -0,0 +1,33 @@
+name: Continuous Integration - Docs
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/docs.yml
+      - "docs/**"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Setup mdBook
+        uses: peaceiris/actions-mdbook@v1
+        with:
+          mdbook-version: '0.4.2'
+
+      - name: Build book
+        run: mdbook build ./docs
+      
+      - name: Write CNAME file
+        run: "echo 'jen.zerotask.net' > ./docs/book/CNAME"
```

Comment:
i think this whole step should be removed as we dont have i cname for this project (i think?)

Reply:
Yep, definitely!